tools/rostest/src/rostest/rostest_main.py

```diff
@@ -98,6 +98,10 @@ def rostestmain():
     parser.add_option("--results-base-dir", metavar="RESULTS_BASE_DIR",
                       help="The base directory of the test results. The test result file is " +
                            "created in a subfolder name PKG_DIR.")
+    parser.add_option("--port", metavar="PORT",
+                      help="default: random unused port number. Note that specifyin this " +
+                           "currently only works with a single test case. See " +
+                           "https://github.com/ros/ros_comm/pull/614")
```

Comment:
That's why I suggested the following more concise help text before:

    ROS Master port number (default: random unused port)